clusters/build-clusters/01_cluster/machine_config/debug_ctrcfg.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: ContainerRuntimeConfig
+metadata:
+  name: debug
+spec:
+  machineConfigPoolSelector:
+    matchLabels:
+      custom-crio: log-devel-debug
```

Comment:
possible typo: log-level instead of log-devel

Reply:
Updated.